Source/WebCore/page/text-extraction/TextExtraction.cpp

```diff
@@ -1476,6 +1494,12 @@ void handleInteraction(Interaction&& interaction, Page& page, CompletionHandler<
 
         return highlightText(page, WTFMove(interaction.nodeIdentifier), WTFMove(interaction.text), interaction.scrollToVisible, WTFMove(completion));
     }
+    case Action::ScrollBy: {
+        if (interaction.scrollDelta.isZero())
+            return completion(false, "Scroll delta is zero"_s);
+
+        return scrollBy(page, WTFMove(interaction.nodeIdentifier), interaction.scrollDelta, WTFMove(completion));
+    }
```

Comment:
nit: I think you don't need a new scope here, since you aren't defining any variable

Reply:
Good point, fixed!